src/main/java/org/tikv/common/importer/TxnImporterStoreClient.java

```diff
@@ -0,0 +1,167 @@
+/*
+ *
+ * Copyright 2021 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.tikv.common.importer;
+
+import io.grpc.ManagedChannel;
+import io.grpc.stub.StreamObserver;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.tikv.common.AbstractGRPCClient;
+import org.tikv.common.PDClient;
+import org.tikv.common.TiConfiguration;
+import org.tikv.common.exception.GrpcException;
+import org.tikv.common.region.RegionManager;
+import org.tikv.common.region.TiStore;
+import org.tikv.common.util.ChannelFactory;
+import org.tikv.kvproto.ImportSSTGrpc;
+import org.tikv.kvproto.ImportSstpb;
+import org.tikv.kvproto.Kvrpcpb;
+
+public class TxnImporterStoreClient
```

Comment:
Can we merge these two classes `TxnImporterStoreClient` and `ImporterStoreClient`?
The logic is almost the same.

Reply:
Maybe we should abstract the response? Because `ImporterStroeClient` implements `StreamObserver<ImportSstpb.RawWriteResponse>` while `TxnImporterStoreClient` implements `StreamObserver<ImportSstpb.WriteResponse>`.